build.ps1

```diff
@@ -287,26 +286,24 @@ function VSX
         [string] $Toolchain
     )
 
-    if($Toolchain -eq 'v110' -and $env:VS110COMNTOOLS -eq $null) {
+    if ($Toolchain -eq 'v110' -and $env:VS110COMNTOOLS -eq $null) {
         Warn "Toolchain $Toolchain is not installed on your development machine, skipping build."
         Return
     }
 
-    if($Toolchain -eq 'v120' -and $env:VS120COMNTOOLS -eq $null) {
+    if ($Toolchain -eq 'v120' -and $env:VS120COMNTOOLS -eq $null) {
         Warn "Toolchain $Toolchain is not installed on your development machine, skipping build."
         Return
     }
 
-    if($Toolchain -eq 'v140' -and $env:VS140COMNTOOLS -eq $null) {
+    if ($Toolchain -eq 'v140' -and $env:VS140COMNTOOLS -eq $null) {
         Warn "Toolchain $Toolchain is not installed on your development machine, skipping build."
         Return
     }
 
     Write-Diagnostic "Starting to build targeting toolchain $Toolchain"
 
-    Msvs "$Toolchain" 'Debug' 'x86'
```

Comment:
From memory they're currently required for a debug build of `CefSharp`, are you sure this is wise?

Reply:
You are right here also. I _thought_ it wasn't needed, but it very much is (since the `/MTd` etc flags need to match for the C++ projects.)

Will revert this also.